.github/workflows/pre-compile_llvm.yml

```diff
@@ -3,26 +3,23 @@ name: Pre-compile llvm
 on:
   workflow_dispatch:
   push:
-    branches: [ release_100, release_11x, release_12x, release_13x ]
+    branches: [ release_100, release_11x, release_12x, release_13x]
 
 jobs:
   build:
+    env:
+      cpp: "g++"
```

Comment:
Remove this and use `echo "cpp=g++" >> $GITHUB_ENV`in the appropriate steps below.

Reply:
fixed